cinema/views.py

```diff
@@ -0,0 +1,35 @@
+from django.http import HttpRequest, HttpResponse, JsonResponse
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+@api_view(["GET", "POST"])
+def movies_list(request) -> HttpResponse:
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
```

Comment:
Check method explicitly before performing an operation, make sure that it is POST before returning a response

Reply:
how about this one?